snapcraft/plugins/ant.py

```diff
@@ -38,21 +53,52 @@
 
 class AntPlugin(snapcraft.plugins.jdk.JdkPlugin):
 
+    @classmethod
+    def schema(cls):
+        schema = super().schema()
+        schema['properties']['ant-properties'] = {
+            'type': 'object',
+            'default': {},
+        }
+        schema['properties']['ant-build-target'] = {
+            'type': 'string',
```

Comment:
or should it be (either by multiple calls from a list or in a oneliner through a ' '.join(...))